tests/test_data/argument_file/args_nested.txt

```diff
@@ -1,3 +1,3 @@
 --include some_rule
--A other_file.txt
+-A args_nested.txt
```

Comment:
These changes are in the scope of another PR #774 and should be removed from here.